server/mediaserver.go

```diff
@@ -187,7 +194,30 @@ func createRTMPStreamIDHandler(s *LivepeerServer) func(url *url.URL) (strmID str
 			mid, key = parseManifestID(resp.ManifestID), resp.StreamKey
 			// Process transcoding options presets
 			if len(resp.Presets) > 0 {
-				presets = parsePresets(resp.Presets)
+				profiles = parsePresets(resp.Presets)
+			}
+
+			if len(resp.Profiles) > 0 {
+				for _, profile := range resp.Profiles {
+					name := profile.Name
+					if name == "" {
+						profile.Name = "webhook_" + common.DefaultProfileName(
+							int(profile.Width),
```

Comment:
cast not needed

Reply:
Assign to `name` rather than `profile.Name`